specs/signature-envelope-cose.md

```diff
@@ -56,6 +65,18 @@ Example of the Notary Project signature payload:
 }
 ```
 
+Example of the Notary Project detached signature payload:
+
+```jsonc
+{
+  "targetArtifact": {
+    "mediaType": "application/octet-stream",
+    "digest": "sha256:2f3a23b6373afb134ddcd864be8e037e34a662d090d33ee849471ff73c873345",
+    "size": 1024
```

Comment:
Do we support `annotation` in the payload? I remember  in this [PR](https://github.com/notaryproject/notation/pull/811/files#diff-30187fc97f8fa76aa528babf8e93d521e4b412bbe47dea781678c6d4d96088e1) , 'notation blob sign --user-metadatra` will add annotation to the signature blob